src/styles/main.scss

```
@@ -2,6 +2,385 @@
+      width: 187px;
+    }
+  }
+}
+
+.menu {
+  background-color: #e5e5e5;
+  &__content {
+    box-sizing: border-box;
+    padding: $headerTopPadding 0;
+    height: 100vh;
+  }
+
+  &__top {
+    margin-bottom: 44px;
+  }
+
+  &__list {
+    list-style: none;
+    margin: 0 0 44px;
+    padding: 0;
+  }
+
+  &__item {
+    margin-bottom: 24px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    padding-bottom: 8px;
+    border-bottom: 1px solid transparent;
+
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 22px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    color: #1d1d1d;
+    text-decoration: none;
+
+    transition: border-color 0.5s;
+
+    &:hover {
+      border-color: #1d1d1d;
+    }
+  }
+
+  &__call {
+    text-decoration: none;
+    display: block;
+    width: max-content;
+
+    &--phone {
+      font-family: "Inter", sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 27px;
+      color: #1d1d1d;
+      margin-bottom: 16px;
+      border-bottom: 1px solid transparent;
+      transition: border-color 0.5s;
+
+      &:hover {
+        border-color: #1d1d1d;
+      }
+    }
+
+    &--action {
+      font-family: "Inter", sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 12px;
+      line-height: 16px;
+      letter-spacing: 2px;
+      text-transform: uppercase;
+
+      color: #1d1d1d;
+      padding-bottom: 8px;
+      border-bottom: 1px solid #1d1d1d;
+    }
+  }
+}
+
+.footer {
+  height: 200px;
+  background-image: url(../images/photos/footer.jpg);
+  background-size: cover;
+  background-position: center;
+
+  @include onTablet {
+    height: 480px;
+  }
+
+  @include onDesktop {
+    height: 555px;
+  }
+}
+
+// main {
+
+// }
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 20px;
+
+  @include onTablet {
+    grid-template-columns: repeat(6, 1fr);
+    column-gap: 20px;
+  }
+
+  @include onDesktop {
+    grid-template-columns: repeat(12, 1fr);
+    column-gap: 24px;
+  }
+
+  &--desktop {
+    display: block;
+
+    @include onTablet {
+      display: grid;
+    }
+
+    @include onDesktop {
+      display: grid;
+    }
+  }
+
+  &__item {
+    @include onDesktop {
+      @for $start from 1 through 12 {
+        @for $end from $start through 12 {
+          &--#{$start}-#{$end} {
+            grid-column: #{$start} / #{$end + 1};
+          }
+        }
+      }
+    }
+
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 140%;
+    color: #fff;
+  }
+
+  &__item-tablet {
+    @include onTablet {
+      @for $start from 1 through 6 {
+        @for $end from $start through 6 {
+          &--#{$start}-#{$end} {
+            grid-column: #{$start} / #{$end + 1};
+          }
```

```suggestion
  &__item {
    @include onTablet {
      @for $start from 1 through 6 {
        @for $end from $start through 6 {
          &-tablet--#{$start}-#{$end} {
            grid-column: #{$start} / #{$end + 1};
          }
```
ти для tablet генеруєш навіть на десктопі зараз